### PR TITLE
Integrating GEISA databank  in the radis interface

### PR DESCRIPTION
**Pull request by arunavabasu-03**
Hey @suzil @erwanp, please check I added the supported molecules. And I will then redirect the users so that see the data also. 

- [x]  Adding the molecules 
- [x] Option to change the databank (Radi's interface)
- [ ]  Fetch the data from geisa

### CODE REVIEW

**Review by erwanp — CHANGES_REQUESTED** [review]
Can you capitalize, trim (remove whitespace) and sort them as in [molecules.ts](https://github.com/suzil/radis-app/blob/9f068e4236e4517e8c02337e5c80f74d3ddec1d5/website/src/components/fields/MoleculeSelector/molecules.ts) ?

**Comment by arunavabasu-03** [comment]
ok @erwanp

**Comment by suzil** [comment]
@arunavabasu-03 Just noticed some `package-lock.json` files. If you use `yarn` instead of `npm`, these extra files shouldn't be created.

**Comment by arunavabasu-03** [comment]
@suzil, I resolve that thing.

File: website/src/components/fields/DatabankSelector/options.tsx [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+      <FormControl fullWidth>
+        <InputLabel id="demo-simple-select-label">Age</InputLabel>
+        <Select
+          labelId="demo-simple-select-label"
+          id="demo-simple-select"
+          // value={age}
+          label="Age"
+          // onChange={handleChange}
+        >
+          <MenuItem value={10}>Ten</MenuItem>
```
Comment (erwanp) [inline website/src/components/fields/DatabankSelector/options.tsx:16]: @arunavabasu-03 are you sure of these Item names? It looks copy paste but not modified?

Comment (arunavabasu-03) [reply website/src/components/fields/DatabankSelector/options.tsx:16]: @erwanp, I resolve the name basically I am working on  testing for the components 😀

**Comment by suzil** [comment]
> @suzil, I resolve that thing.

Would you remove the files `package-lock.json` and `website/package-lock.json`? I still see them in the files changed for this PR.

**Comment by erwanp** [comment]
Radis-0.13 is released, so it's now possible to compute GEISA; using `databank='geisa'` instead of `databank='hitran'`

**Comment by erwanp** [comment]
cf #570

**Pull request closed without merging**